### PR TITLE
Hero Section Update

### PR DESCRIPTION
**Pull request by zediogoviana**
Hero Section Update

Why:

- Update section for new design version

How:

- Added new copy component for the copy.

- Continuous timed roll and click change of copy message.

### CODE REVIEW

File: src/components/HeroCopyElement/index.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+          aria-label="clickable info 3"
+          styleName={getClassName(2)}
+          style={{ "--top": "2" }}
+          onClick={() => reset(2)}
+        />
+      </div>
+      <div styleName="message">
+        <Typography color="oxford-blue">{textPlaceholder[position]}</Typography>
+        <Typography color="klein-blue" weight="bold" variant="h2">
+          We materialize ideas
```
Comment (taniaprocha) [inline src/components/HeroCopyElement/index.js:72]: This text also changes

File: src/components/HeroCopyElement/index.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    } else if (!isActive && seconds !== 0) {
+      clearInterval(interval);
+    }
+
+    return () => clearInterval(interval);
+  }, [isActive, seconds]);
+
+  return (
+    <div styleName="copy-section">
+      <div styleName="lines">
```
Comment (taniaprocha) [inline src/components/HeroCopyElement/index.js:46]: I think we need to have a higher clickable area for the line

File: src/components/HeroCopyElement/index.js (hunk 1 above)
Comment (zediogoviana) [reply src/components/HeroCopyElement/index.js:72]: Oops! Did not see that... It's a quick update, I'll wait for this deploy to finish so we can discuss animations, and then add these changes also.

File: src/components/HeroCopyElement/index.js (hunk 2 above)
Comment (zediogoviana) [reply src/components/HeroCopyElement/index.js:50]: What do you suggest?

Comment (taniaprocha) [reply src/components/HeroCopyElement/index.js:41]: have a white background area behind the line

Comment (zediogoviana) [reply src/components/HeroCopyElement/index.js:41]: Check my latest commit, I think it solved it... But instead, I used 20px

File: src/components/HeroCopyElement/index.css [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  outline: none;
+}
+
+.line-baby-blue {
+  --top: 0;
+  position: absolute;
+  top: calc($spacing-52 / 2 + var(--top) * $spacing-32);
+
+  width: 86px;
+  height: 20px;
```
Comment (taniaprocha) [inline src/components/HeroCopyElement/index.css:73]: use `$spacing-20`

File: src/components/HeroCopyElement/index.js (hunk 2 above)
Comment (taniaprocha) [reply src/components/HeroCopyElement/index.js:41]: It's ok now

File: src/components/HeroCopyElement/index.css [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+  cursor: pointer;
+  outline: none;
+}
+
+.line-baby-blue {
+  --top: 0;
+  position: absolute;
+  top: calc($spacing-52 / 2 + var(--top) * $spacing-32);
+
+  width: 86px;
```
Comment (taniaprocha) [inline src/components/HeroCopyElement/index.css:72]: should be `$column`

File: src/sections/HomePortfolio/index.js [hunk 5]
```diff
@@ -9,7 +9,6 @@ import withQuery from "./withQuery";
 
 import "./index.css";
 
-
```
Comment (taniaprocha) [inline src/sections/HomePortfolio/index.js:12]: Do we need this new line?

Comment (zediogoviana) [reply src/sections/HomePortfolio/index.js:12]: No, it's redundant. That was probably my linter that deleted the line. Should I revert it?

Comment (taniaprocha) [reply src/sections/HomePortfolio/index.js:12]: yes please

File: src/components/HeroCopyElement/index.css [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+  }
+}
+
+.line-klein-blue {
+  --top: 0;
+  position: absolute;
+  top: calc($spacing-52 / 2 + var(--top) * $spacing-32);
+
+  width: 118px;
+  height: 20px;
```
Comment (taniaprocha) [inline src/components/HeroCopyElement/index.css:59]: Also `$spacing-20`

**Pull request merged**